invenio_cli/helpers/cookiecutter_wrapper.py

```diff
@@ -23,12 +23,34 @@
 class CookiecutterWrapper(object):
     """Cookiecutter helper object for InvenioCLI."""
 
-    def __init__(self, flavour):
-        """Constructor."""
+    @classmethod
+    def extract_template_name(cls, template):
+        """Extract template name from a template URL."""
+        name = template.rstrip('/').rpartition("/")[2]
+        git = '.git'
+        if name.endswith(git):
+            name = name[:-len(git)]
+        return name
+
+    def __init__(self, flavour, cookiecutter):
+        """Constructor.
+
+        :param flavour: "RDM" or something else: String
+        :param cookiecutter: tuple of (template: URL, checkout: String)
+        """
         self.tmp_file = None
         self.template_name = None
         self.flavour = flavour
 
+        if self.flavour.upper() == 'RDM':
+            self.template = (
+                cookiecutter[0] or
+                'https://github.com/inveniosoftware/'
+                'cookiecutter-invenio-rdm.git'
+            )
+            self.template_name = self.extract_template_name(self.template)
```

Comment:
Where is the `template_name` used/needed?

Reply:
It's used on the replay part. It does not appear here cuz there is no changes, if you open the full file is at the bottom :)

Reply:
What he said :)